.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: c
+sudo: false
+
+cache:
+  directories:
+  - $HOME/.ghc
+  - $HOME/.stack
+
+matrix:
+  include:
+  - env: GHCVER=7.10.3 CABALVER=1.22
+    addons: {apt: {packages: [cabal-install-1.22,ghc-7.10.3], sources: [hvr-ghc]}}
+
+before_install:
+  - mkdir -p ~/.local/bin
+  - export PATH=~/.local/bin:$PATH
+  - curl -L https://www.stackage.org/stack/linux-x86_64 | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
```

Comment:
I changed it to download a specific version of stack.

From looking at how other people set up their builds, I'm not convinced an extra verification of the sha1's would be that useful...both artifacts are being downloaded from github, and over https.